### ISSUE TITLE
Code Table Request - event method

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#6218]
## Instructions

_This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions._

_Reviewing documentation before proceeding will result in a more enjoyable experience._

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

For background see #5703

_**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

event method

_**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

~~Description of how the event was carried out.~~
Description of methods applied at the event.

_**Attribute data type** If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)

free-text

_**Attribute controlled values** If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate.

N/A

_**Attribute units** If numberical values should be accompanied by units, provide a link to the appropriate units table.

N/A

_**Context**: Describe why this new value is necessary and existing values are not._

_**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcoll_event_attr_type

_**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

N/A

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

_**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions._

yes

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval - https://github.com/ArctosDB/arctos/issues/6218#issuecomment-1540270331 
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by dustymc** [comment]
1. "event"  - I like it (it's clear/unambiguous) and hate it (anything attached to an event must be an event-whatever!). Are we sure that this is necessary?
2. collecting - I think this must just be wrong, at least it's overly-dependent. (Surely method could apply to biopsies and photography and - lots of things that don't involve "collecting" as we've defined it in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type
3. Description of how the cataloged item was obtained.  - seems (somehow) overly constrained - 'what happened at the event'??

**Comment by campmlc** [comment]
I was kind of thinking the same thing. Cultural collections may want different language. "Event method" ?

**Comment by DerekSikes** [comment]
Just chiming in to ask again about this. If I start using this event-method attribute will it be mapped to the Darwin core collection method so GBIF et al. get the data?

**Comment by dustymc** [comment]
>mapped to the Darwin core

That should be a separate issue after this is finalized but sure, I can map whatever wherever.

**Comment by Jegelewicz** [comment]
> I think this must just be wrong, at least it's overly-dependent. (Surely method could apply to biopsies and photography and - lots of things that don't involve "collecting" as we've defined it in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type

Is it? or do we (or will we) just need different methods? creation method, etc?

If we want to have one attribute for them all, then perhaps it should be event method?

Description of methods applied at the event.

**Comment by AJLinn** [comment]
> Is it? or do we (or will we) just need different methods? creation method, etc?

Creation is creation... not sure what our methods would be that could be summarized in a single term.
Use is the method... 

> If we want to have one attribute for them all, then perhaps it should be event method?

I prefer this, as I've never really understood the distinction between "collecting event" and "specimen event" given our particular kinds of events.

**Comment by Jegelewicz** [comment]
> I've never really understood the distinction between "collecting event" and "specimen event" given our particular kinds of events.

I am not sure that very many people understand this, so here goes:

"specimen" or catalog record event is the data that ties an event "collecting event" to a catalog record:

The "specimen event" consists of

[Type](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type) - for the object or observation in the catalog record, what happened at this event?
Assigned By - agent who said that the event should be associated with this catalog record
Assigned date - date the agent said that
[Verification Status](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctverificationstatus) - A measure of certainty about the fact that this event should be associated with this catalog record
Verified By - agent who made the verification status assertion
Verified Date - Date they made the verification status assertion
Collecting Method - for events of type "collection" describes how the collection took place
[Collecting Source](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcollecting_source) - for events of type "collection" describes the source of the collected object(s)
Habitat - describes the environment where the event too place
Associated Species - a list of other species observed/collected at the event
Event Remark - any remarks about why this event should be associated with this catalog record

All other "event" stuff goes in the event ("collecting event"). This is basically, verbatim information, date(s), and locality plus a bunch of attributes (that also include collecting method and habitat, because reasons...).

Using this method means that a single event ("collecting event") could be a collection event for one thing and a creation for another - cool! It also highlights the fact that these things are poorly labeled and even more poorly understood. I hope the above is all correct, because it is how we are re-naming bulkloader columns.

**Comment by campmlc** [comment]
But when we despecimen the event language, we are using record event for "specimen event", correct? Maybe collecting  event should just be " event", with record event trying records to events, and we get rid of "collecting" ?

**Comment by campmlc** [comment]
I support this.

**Comment by Jegelewicz** [comment]
@ArctosDB/arctos-code-table-administrators this just needs one more box checked. Is anyone willing or do we think this needs more discussion?

**Comment by campmlc** [comment]
I've already approved, so we need someone else. @Nicole-Ridgwell-NMMNHS ?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Looks good to me! Checkmark added.

**Comment by Jegelewicz** [comment]
Move all current record_collecting_method to this new attribute.

Change the name of collecting event method to "record_specific collection method"

@dustymc we need numbers on how many events are shared by catalog records that have different collecting methods.

**Comment by Jegelewicz** [comment]
But make this attribute then we can discuss migration.

**Comment by Jegelewicz** [comment]
Added and associated issues created.
